### PR TITLE
Build new backstage plugin for Azure Container Registry

### PR DESCRIPTION
**Pull request by yzhao583**
Story: [https://github.com/janus-idp/backstage-plugins/issues/216](https://github.com/janus-idp/backstage-plugins/issues/216)

Functionality:
- Fetch and display tags information for a [repository](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-concepts) from Azure Container Registry.

Screenshot:
Screenshot 2023-04-28 at 1 27 11 PM

### CODE REVIEW

File: plugins/acr/package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@janus-idp/plugin-acr",
```
Comment (tumido) [inline plugins/acr/package.json:2]:
```suggestion
  "name": "@janus-idp/backstage-plugin-acr",
```

File: plugins/acr/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```
Comment (tumido) [inline plugins/acr/package.json:7]:
```suggestion
```

**Review by tumido — CHANGES_REQUESTED** [review]
Hey! Great start! Please mind we won't be able to do a proper technical review on a new plugin this week (due to our other commitments and deadlines). I've pointed out a few main details that have to be changed in the `package.json`. I'll try to get back to you with a proper technical review as soon as possible.

**Comment by invincibleJai** [comment]
closing this in favour of https://github.com/janus-idp/backstage-plugins/pull/483

**Pull request closed without merging**